src/charm.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""GLAuth Operator Charm."""
```

Comment:
Just a stylistic suggestion to add a newline between the license header and the module-level docstring.

```suggestion

"""GLAuth Operator Charm."""
```